tools/addrundata/rundata.go

```diff
@@ -92,29 +103,22 @@ func parseInit(sc *bufio.Scanner) (*mpb.Metadata, error) {
 	return nil, errors.New("func init() was not terminated")
 }
 
-var tmpl = template.Must(template.New("rundata_test.go").Parse(
-	`// Code generated by go run tools/addrundata; DO NOT EDIT.
-package {{.Package}}
-
-import "github.com/openconfig/featureprofiles/internal/rundata"
+var tmpl = template.Must(template.New("metadata.textproto").Parse(
+	`# proto-file: proto/metadata.proto
```

Comment:
For proto-file, use fully qualified path github.com/openconfig/featureprofiles/proto/metadata.proto?

Reply:
Yes, that does look like the norm, thanks.